api/src/Entity/Camp.php

```diff
@@ -65,7 +65,7 @@
 #[ORM\UniqueConstraint(name: 'couponKey_unique', columns: ['couponKey'])]
 class Camp extends BaseEntity implements BelongsToCampInterface, CopyFromPrototypeInterface {
     public const ITEM_NORMALIZATION_CONTEXT = [
-        'groups' => ['read', 'Camp:Periods', 'Period:Days', 'Camp:CampCollaborations', 'CampCollaboration:User'],
+        'groups' => ['read', 'Camp:Periods', 'Period:Days', 'Camp:CampCollaborations', 'CampCollaboration:User', 'User:Profile'],
```

Comment:
Revert this and create a filter by camp or collaboration or user on the profiles endpoint